tests/database/test_chaindb.py

```diff
@@ -75,11 +75,22 @@ def test_persist_block_to_db(chaindb, block):
     assert chaindb.exists(block_to_hash_key)
 
 
-def test_get_score(chaindb, block):
-    chaindb.persist_block_to_db(block)
-    block_to_hash_key = make_block_hash_to_score_lookup_key(block.hash)
-    score = rlp.decode(chaindb.db.get(block_to_hash_key), sedes=rlp.sedes.big_endian_int)
-    assert chaindb.get_score(block.hash) == score
+def test_get_score(chaindb):
+    genesis = BlockHeader(difficulty=1, block_number=0, gas_limit=0)
+    chaindb.persist_header_to_db(genesis)
+
+    genesis_score_key = make_block_hash_to_score_lookup_key(genesis.hash)
+    genesis_score = rlp.decode(chaindb.db.get(genesis_score_key), sedes=rlp.sedes.big_endian_int)
+    assert genesis_score == 1
+    assert chaindb.get_score(genesis.hash) == 1
+
+    block1 = BlockHeader(difficulty=10, block_number=1, gas_limit=0, parent_hash=genesis.hash)
+    chaindb.persist_header_to_db(block1)
+
+    block1_score_key = make_block_hash_to_score_lookup_key(block1.hash)
```

Comment:
Not something that needs to be cleaned in this PR but:

Testing the value directly retrieved from the database feels like testing the implementation vs testing the behavior which typically results in fragile tests.  I suspect that we'll want to run these tests against multiple database backends at some point at which point this test will need to be re-worked.

No real question or request there.  Just a brain dump.

Reply:
The original test had this, and I believed it was because you wanted to check that the score was stored under the correct key in the DB, so I kept it. I'd be happy to drop that and rely only on asserting the result of `.get_score()` is what we expect

Reply:
nah, lets punt.  It'll fail and make us change it as soon as we start supporting more than one chainDB or database backend or whatever it is that makes this no-longer-appropriate and since we don't know exactly what that's going to look like, I think it's best to wait.